diff --git a/node_modules/react-native-reanimated/src/reanimated2/core.js b/node_modules/react-native-reanimated/src/reanimated2/core.js
index 273bd1d..d6e8f7c 100644
--- a/node_modules/react-native-reanimated/src/reanimated2/core.js
+++ b/node_modules/react-native-reanimated/src/reanimated2/core.js
@@ -255,5 +255,5 @@ runOnUI(() => {
     error: runOnJS(capturableConsole.error),
     info: runOnJS(capturableConsole.info),
   };
-  _setGlobalConsole(console);
+  // _setGlobalConsole(console);
 })();
